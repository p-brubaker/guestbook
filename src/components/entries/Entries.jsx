import styles from './Entries.module.css';
import React from 'react';

export default function Entries({ guests }) {
  return (
    <div className="entries-container">
      {guests.map((guest) => (
        <div className={styles.entry} key={guest.id}>
          <p>{guest.name}</p>
          <p>{guest.entry}</p>
        </div>
      ))}
    </div>
  );
}
