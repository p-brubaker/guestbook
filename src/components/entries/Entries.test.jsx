import { render } from '@testing-library/react';
import Entries from './Entries';
import React from 'react';

const mockGuests = [
  {
    id: 1,
    name: 'Blob',
    entry: 'Hi!',
  },
];

it('should render the Entries component', async () => {
  const { container } = render(<Entries guests={mockGuests} />);

  expect(container).toMatchSnapshot();
});
