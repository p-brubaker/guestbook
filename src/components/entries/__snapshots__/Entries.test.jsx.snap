// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Entries component 1`] = `
<div>
  <div
    class="entries-container"
  >
    <div
      class="entry"
    >
      <p>
        Blob
      </p>
      <p>
        Hi!
      </p>
    </div>
  </div>
</div>
`;
